src/core/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -13,32 +13,39 @@
-                    <RadioButton Content="Dark"/>
-                    <RadioButton Content="Light"/>
-                    <RadioButton Content="System default" IsChecked="True"/>
+                    <RadioButton x:Name="Rodio_Theme_Dark" Content="Dark" Tag="Dark" Checked="Theme_Changed"/>
+                    <RadioButton x:Name="Rodio_Theme_Light"  Content="Light" Tag="Light" Checked="Theme_Changed"/>
+                    <RadioButton x:Name="Rodio_Theme_Default"  Content="System default" Tag="System" Checked="Theme_Changed"/>
                 </muxc:RadioButtons>
 
+                <!--
                 <ToggleSwitch Header="Disable telemetry" IsOn="True" Margin="0,14,0,0" />
                 <TextBlock Text="PowerToys currently respects the Windows data &amp; feedback setting" Opacity="0.8" Margin="0,0,0,0" />
+                    -->
 
+                <Button Background="{ThemeResource SystemAccentColor}" Content="Restart as admin" Margin="0,10,0,0" Foreground="White" Click="Restart_Elevated" />
+
+                <!--
                 <TextBlock Text="Default apps" Style="{StaticResource SubtitleTextBlockStyle}" Margin="0,34,0,8"/>
                 <ComboBox Header="Shell" SelectedIndex="0" MinWidth="240" Margin="0,14,0,0">
                     <ComboBoxItem>PowerShell</ComboBoxItem>
                 </ComboBox>
                 <ComboBox Header="Terminal" SelectedIndex="0" MinWidth="240" Margin="0,14,0,0">
                     <ComboBoxItem>Windows Console</ComboBoxItem>
                 </ComboBox>
+               -->
+
                 <TextBlock Text="About PowerToys" Style="{StaticResource SubtitleTextBlockStyle}" Margin="0,34,0,8"/>
                 <TextBlock FontWeight="Bold" Text="Version 0.15.1.0" Margin="0,14,0,0" />
-                <Button Background="{ThemeResource SystemAccentColor}" Content="Check for updates" Margin="0,10,0,0" Foreground="White"/>
-                <!-- TO DO: The styling of this button should be improved so the hover/pressed states are representing the SystemAccentColor -->
+                <!--<Button Background="{ThemeResource SystemAccentColor}" Content="Check for updates" Margin="0,10,0,0" Foreground="White" />
+                 TO DO: The styling of this button should be improved so the hover/pressed states are representing the SystemAccentColor -->
 
-                <HyperlinkButton Content="Report a bug" Margin="0,14,0,0" />
-                <HyperlinkButton Content="Request a feature"  />
-                <HyperlinkButton Content="Privacy statement"  />
+                <HyperlinkButton Content="Report a bug" Margin="0,14,0,0" NavigateUri="https://github.com/microsoft/PowerToys/issues" />
+                <HyperlinkButton Content="Request a feature"  NavigateUri="https://github.com/microsoft/PowerToys/issues"/>
+                <HyperlinkButton Content="Privacy statement" NavigateUri="https://privacy.microsoft.com/en-us/privacystatement" />
```

Comment:
perhaps add TODO for a localized privacy link?

Reply:
Think if we drop the en-us, it should auto correct.  Can you do a quick test?

Reply:
Still redirected me to `en-us`. I have Russian IP but use English Windows.

Reply:
UPD: on Russian Windows it correctly redirects to a localized page.

Reply:
@yuyoyuppe does http://go.microsoft.com/fwlink/?LinkId=521839 go to a correctly localized page?

Reply:
Yes, it works - same behavior as https://privacy.microsoft.com/privacystatement. Of course it's correct that it shows the English version while on English Windows.

Reply:
part of me likes having a non-obfusicated link.  But good to know it behaves correctly.  lets go with the go link since it properly localizes stuff.